src/scss/components/optional/_overflow-carousel.scss

```diff
@@ -97,6 +97,7 @@
 
     .c-overflowCarousel--hideScroll {
         @include media('>=wide') {
+         /* stylelint-disable-next-line property-no-unknown */
```

Comment:
Is this indentation ok?

Reply:
If you can fix it, I probably would. And while you're at it could you bump the date in the changelog, please!

Reply:
I was more asking about what the best practice is, should the comment line up with the @include?